src/core/services/layout/layout.js

```diff
@@ -239,4 +290,35 @@
 
     }
 
+    /**
+     * Scan the body element. If it has a class 'md-css-only', then do NOT
+     * postLink process the directives for Attribute selectors.
+     * (recall that postlink injects Class selectors based on attribute selector settings)
+     *
+     * Instead the Layout CSS for Attributes is used:
+     * e.g
+     *       .md-css-only [layout=row] {
+     *          flex-direction: row;
+     *          -webkit-flex-direction: row;
+     *       }
+     *
+     * Note: this means that 'md-css-only' will not work for IE (due to performance issues)
+     * In these cases, the Layout translators (directives) should be enabled and the
+     * `angular-material.[min.]js` must be loaded.
+     */
+    function postLinkIsDisabled(document) {
+      var disablePostLinks = $$mdLayout.disablePostLinks;
+
+      // Perform a read-once (1x) check for the `md-css-only` class on the BODY
+
+      if ( angular.isUndefined(disablePostLinks) ) {
```

Comment:
The possible values are true, false, undefined/null. Only if undefined/null, do we scan the body attribute value.